common/src/main/java/dev/architectury/event/events/common/LifecycleEvent.java

```diff
@@ -88,6 +89,14 @@
      * @see ServerLevelState#act(Level)
      */
     Event<ServerLevelState> SERVER_LEVEL_SAVE = EventFactory.createLoop();
+    /**
+     * Invoked once common setup has begun.
+     * <p> This happens during {@code FMLCommonSetupEvent} on Forge,
+     * or when Architectury API's entrypoint initialises on Fabric.
+     * <p>
+     * Registries should be initialised here.
```

Comment:
To a new user, "Registries should be initialised here." implies that **they** should initialise their registries here, which would be incorrect. This should probably be rephrased to "Registries should have been initialised by this point."